crates/re_renderer/src/context.rs

```diff
@@ -106,6 +123,32 @@ impl RenderContext {
     ) -> Self {
         re_tracing::profile_function!();
 
+        let frame_index_for_uncaptured_errors = Arc::new(AtomicU64::new(STARTUP_FRAME_IDX));
+
+        // Make sure to catch all errors, never crash and deduplicate reported errors.
+        // `on_uncaptured_error` is a last-resort handler which we should never hit,
+        // since there should always be an open error scope.
+        //
+        // Note that this handler may not be called for all errors if no scope is present.
+        // (as of writing, wgpu-core will always call it when there's no scope, but Dawn doesn't!)
+        // Therefore, it is important to always have a scope open!
+        // See https://www.w3.org/TR/webgpu/#telemetry
+        let top_level_error_tracker = {
+            let err_tracker = std::sync::Arc::new(ErrorTracker::default());
```

Comment:
This file is inconsistent with `std::sync::Arc` vs just `Arc`. I suggest the shorter one (everyone knows what an `Arc` is - it should be in the std prelude imho)